modules/delegates.js

```diff
@@ -272,25 +281,30 @@ __private.checkDelegates = function (publicKey, votes, state, cb) {
  * @returns {setImmediateCallback} 
  */
 __private.loadDelegates = function (cb) {
-	var secrets;
+	var secretsList;
 
 	if (library.config.forging.secret) {
 		if (Array.isArray(library.config.forging.secret)) {
-			secrets = library.config.forging.secret;
+			secretsList = library.config.forging.secret;
 		} else {
-			secrets = [library.config.forging.secret];
+			secretsList = [library.config.forging.secret];
 		}
 	}
 
-	if (!secrets || !secrets.length) {
+	if (!secretsList || !secretsList.length || !library.config.forging.force) {
 		return setImmediate(cb);
 	} else {
-		library.logger.info(['Loading', secrets.length, 'delegates from config'].join(' '));
+		library.logger.info(['Loading', secretsList.length, 'delegates from config'].join(' '));
 	}
 
-	async.eachSeries(secrets, function (secret, cb) {
+	async.eachSeries(secretsList, function (encryptedItem, cb) {
+		var secret = __private.decryptSecret(encryptedItem.encryptedSecret, library.config.forging.defaultKey);
```

Comment:
Main goal is to not allow sending secrets over the network. We can stay with current approach - it's good. I just like flexibility. ;)